glamour.go

```diff
@@ -135,20 +135,18 @@ func WithEnvironmentConfig() TermRendererOption {
 // standard style.
 func WithStylePath(stylePath string) TermRendererOption {
 	return func(tr *TermRenderer) error {
-		jsonBytes, err := ioutil.ReadFile(stylePath)
-		switch {
-		case err == nil:
-			return json.Unmarshal(jsonBytes, &tr.ansiOptions.Styles)
-		case os.IsNotExist(err):
-			styles, err := getDefaultStyle(stylePath)
-			if err != nil {
+		styles, err := getDefaultStyle(stylePath)
+		if err != nil {
+			jsonBytes, err := ioutil.ReadFile(stylePath)
+			switch {
```

Comment:
Maybe we should drop the `switch` here and use a simple `if`? :smiley:

Reply:
Ah. Duh. I copy-pasted the existing lines into a new order and didn't even look to see if I could improve it. 😆